src/schema/schema_util.c

```diff
@@ -9,41 +9,56 @@
 #include "wt_internal.h"
 
 /*
- * __wt_schema_backup_check --
- *	Check if a backup cursor is open and give an error if the schema
- * operation will conflict.  This is called after the schema operations
- * have taken the schema lock so no hot backup cursor can be created until
- * this is done.
+ * __schema_backup_check_requires_hot_backup_lock --
+ *	Helper for __wt_schema_backup_check.  Intended to be called while
+ *	holding the hot backup read lock.
  */
-int
-__wt_schema_backup_check(WT_SESSION_IMPL *session, const char *name)
+static int
+__schema_backup_check_requires_hot_backup_lock(
+    WT_SESSION_IMPL *session, const char *name)
```

Comment:
Like the others, just call this the `_int` version of its parent function.

Reply:
bc1914f